.eslintrc

```diff
@@ -7,5 +7,13 @@
   "globals": {
     "expect": true,
     "sinon": true
-  }
+  },
+  "overrides": [
+    {
+      "files": ["test/**/*.js"],
+      "rules": {
+        "no-unused-expressions": "off"
```

Comment:
Please add a comment on why this exclusion.

Reply:
done